pandas/core/series.py

```diff
@@ -4371,8 +4371,9 @@ def memory_usage(self, index=True, deep=False):
             v += self.index.memory_usage(deep=deep)
         return v
 
-    @Appender(generic.NDFrame._take.__doc__)
-    def _take(self, indices, axis=0, is_copy=False):
+    @Appender(generic.NDFrame.take.__doc__)
+    def take(self, indices, axis=0, is_copy=False, **kwargs):
```

Comment:
Do we need to add kwargs here or did you just add this for consistency?

Reply:
I added it for consistency:
`Series` class inherits from the `NDFrame`, so I wanted to preserve the same signature for `take`.